config/config.go

```diff
@@ -59,6 +59,9 @@ type ServerConfig struct {
 	DbSslKeyContents      string `ini:"db_sslkey_contents"`
 	DbUseIamAuth          bool   `ini:"db_use_iam_auth"`
 
+	// Postgres data directory, as used for system stats (autodetected if unset)
+	DataDirectory string `ini:"db_data_directory"`
```

Comment:
Should we call this variable `DbDataDirectory` for consistency?

Reply:
Yeah, good call.